export/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/ExportMessageReceiverTest.java

```diff
@@ -121,7 +125,15 @@ ExportMessageReceiver testStowRsExportMessageReceiver(
     DicomWebClient sourceDicomWebClient =
         new DicomWebClient(fakeSourceDicomWebServer.createRequestFactory(), HttpTesting.SIMPLE_URL);
     DicomWebClient sinkDicomWebClient =
-        new DicomWebClient(fakeSinkDicomWebServer.createRequestFactory(), HttpTesting.SIMPLE_URL);
+        new DicomWebClient(fakeSinkDicomWebServer.createRequestFactory(), HttpTesting.SIMPLE_URL) {
+          public void stowRs(String path, InputStream in) throws IDicomWebClient.DicomWebException {
+            try {
+              requestFactory.buildGetRequest(new GenericUrl("http://nope")).execute();
```

Comment:
For my understanding, this request actually does complete since the server is a FakeWebServer? If so could you add a small comment explaining that?

Reply:
Ok